akka-persistence/src/test/scala/akka/persistence/RecoveryPermitterSpec.scala

```diff
@@ -169,11 +169,6 @@ class RecoveryPermitterSpec extends PersistenceSpec(ConfigFactory.parseString(
       val persistentActor = system.actorOf(testProps("p3", p3.ref, throwFromRecoveryCompleted = true))
       p3.expectMsg(RecoveryCompleted)
       p3.expectMsg("postStop")
-      // it's restarting
-      (1 to 5).foreach { _ ⇒
-        p3.expectMsg(RecoveryCompleted)
-        p3.expectMsg("postStop")
-      }
```

Comment:
Why was this changed? I can't see that something in the implementation was changed for this test case. Here the exception is thrown inside the Receive, not when creating the Receive.

Reply:
Sorry .I had misunderstood.